publicAPI/src/hcode-values/hcode-values.controller.spec.ts

```diff
@@ -1,21 +1,63 @@
-import { HttpModule } from "@nestjs/axios";
-import { Test, TestingModule } from "@nestjs/testing";
+import { HttpModule, HttpService } from '@nestjs/axios';
+import { Test, TestingModule } from '@nestjs/testing';
+import { Observable, of } from 'rxjs';
 
-import { HCodeValuesController } from "./hcode-values.controller";
+import { HCodeValuesService } from '../services/hcode-values/hcode-values.service';
+import { HCodeValuesController } from './hcode-values.controller';
 
 describe("HCodeValuesController", () => {
   let controller: HCodeValuesController;
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
       imports: [HttpModule],
+      providers: [
+        HCodeValuesService,
+        { provide: HttpService, useValue: { get: (url: string) => handleGetRequests(url) } }],
       controllers: [HCodeValuesController],
     }).compile();
 
     controller = module.get<HCodeValuesController>(HCodeValuesController);
   });
 
-  it("should be defined", () => {
+  fit("should be defined", () => {
```

Comment:
Replace `fit` with `it` otherwise only the tests marked with `fit` will be executed